fhir-term-graph/src/main/java/com/ibm/fhir/term/graph/impl/FHIRTermGraphImpl.java

```diff
@@ -0,0 +1,214 @@
+
+        // property keys (not indexed)
+        management.makePropertyKey("count").dataType(Integer.class).make();
+        management.makePropertyKey("group").dataType(String.class).make();
+        management.makePropertyKey("language").dataType(String.class).make();
+        management.makePropertyKey("system").dataType(String.class).make();
+        management.makePropertyKey("use").dataType(String.class).make();
+        management.makePropertyKey("valueDateTime").dataType(String.class).make();
+        management.makePropertyKey("valueDecimalString").dataType(String.class).make();
+
+        // vertex labels
+        management.makeVertexLabel("CodeSystem").make();
+        management.makeVertexLabel("Concept").make();
+        management.makeVertexLabel("Designation").make();
+        management.makeVertexLabel("Property_").make();
+
+        // edge labels
+        management.makeEdgeLabel(FHIRTermGraph.IS_A).make();
+        management.makeEdgeLabel("concept").make();
+        management.makeEdgeLabel("designation").make();
+        management.makeEdgeLabel("property_").make();
+
+        // composite indexes
+        management.buildIndex("byCode", Vertex.class).addKey(code).buildCompositeIndex();
+        management.buildIndex("byCodeLowerCase", Vertex.class).addKey(codeLowerCase).buildCompositeIndex();
+        management.buildIndex("byDisplay", Vertex.class).addKey(display).buildCompositeIndex();
+        management.buildIndex("byUrl", Vertex.class).addKey(url).buildCompositeIndex();
+        management.buildIndex("byUrlAndVersion", Vertex.class).addKey(url).addKey(version).buildCompositeIndex();
+        management.buildIndex("byValue", Vertex.class).addKey(value).buildCompositeIndex();
+
+        management.buildIndex("byValueBoolean", Vertex.class).addKey(valueBoolean).buildCompositeIndex();
+        management.buildIndex("byValueCode", Vertex.class).addKey(valueCode).buildCompositeIndex();
+//      management.buildIndex("byValueDateTime", Vertex.class).addKey(valueDateTime).buildCompositeIndex();
+        management.buildIndex("byValueDateTimeLong", Vertex.class).addKey(valueDateTimeLong).buildCompositeIndex();
+        management.buildIndex("byValueDecimal", Vertex.class).addKey(valueDecimal).buildCompositeIndex();
+        management.buildIndex("byValueInteger", Vertex.class).addKey(valueInteger).buildCompositeIndex();
+        management.buildIndex("byValueString", Vertex.class).addKey(valueString).buildCompositeIndex();
+
+        // mixed indexes
+        management.buildIndex("vertices", Vertex.class)
+            .addKey(display, Mapping.STRING.asParameter())
+            .addKey(value, Mapping.STRING.asParameter())
+            .addKey(valueString, Mapping.STRING.asParameter())
+            .addKey(valueCode, Mapping.STRING.asParameter())
+            .buildMixedIndex("search");
+
+        log.info(System.lineSeparator() + management.printSchema());
```

Comment:
Is this something we always want to print?

Reply:
The schema is only created once so I think it is okay to leave it in.